handler/src/main/java/io/netty/handler/timeout/IdleStateEvent.java

```diff
@@ -21,19 +21,14 @@
  * A user event triggered by {@link IdleStateHandler} when a {@link Channel} is idle.
  */
 public final class IdleStateEvent {
-    public static final IdleStateEvent FIRST_READER_IDLE_STATE_EVENT = new IdleStateEvent(IdleState.READER_IDLE, true);
```

Comment:
@Scottmitch Sorry—I know this has come up before, and I'm worried that I'm missing important guidelines. I totally get why this is an API change and totally understand the consequences of those changes. My understanding was that 4.1 isn't "final" yet, and so there's still a little wiggle room.

For the future, is the concern about doing this in 4.1 generally, or about changing the API for any version? Would it make more sense to propose this against `master`?